lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -358,7 +358,7 @@ def __init__(self, ufeSceneItem):
         self.createMetadataSection()
         cmds.editorTemplate(endScrollLayout=True)
 
-        if int(cmds.about(majorVersion=True)) > 2022:
+        if ('%s.%s' % (cmds.about(majorVersion=True), cmds.about(minorVersion=True))) > '2022.1':
```

Comment:
I don’t understand. You are making greater than operation on strings?

Reply:
Yes because I need to test the Maya update (minor) version. Why? Do you think there is a case where it won't work correctly?

Reply:
Wasn't expecting that to work...but looks like it's working without issues in python. Learned something new.